pkg/webhooks/leaderworkerset_webhook.go

```diff
@@ -67,6 +67,11 @@ func (r *LeaderWorkerSetWebhook) Default(ctx context.Context, obj runtime.Object
 			MaxSurge:       intstr.FromInt32(0),
 		}
 	}
+
+	if lws.Spec.NetworkConfig == nil {
```

Comment:
we need to default `lws.Spec.NetworkConfig.SubdomainPolicy` in case networkconfig was set but not the policy. `SubdomainPolicy` should be a pointer as mentioned in the api file.

Reply:
fixed